experimental/python/databricks/bundles/build.py

```diff
@@ -446,14 +446,14 @@ def main(argv: list[str]) -> int:
     logging.basicConfig(level=logging.DEBUG, stream=sys.stderr)
     new_bundle, locations, diagnostics = python_mutator(args)
 
-    with open(args.diagnostics, "w") as f:
+    with open(args.diagnostics, "w", encoding="utf-8") as f:
         _write_diagnostics(f, diagnostics)
 
     if locations_path := args.locations:
-        with open(locations_path, "w") as f:
+        with open(locations_path, "w", encoding="utf-8") as f:
             _write_locations(f, locations)
 
-    with open(args.output, "w") as f:
+    with open(args.output, "w", encoding="utf-8") as f:
```

Comment:
Rather than handling this case-by-case, should we set locale-related env vars so that everything in Python process (including libraries and subprocess) works with unicode?

Reply:
I think it can be weird if we start customizing the Python environment. For example, if customers write unit tests, they will have a different behaviour. We don't have any 3p dependencies and IO is very limited, so I think we should be just carefully specifying encoding where needed.